### DISCUSSION TITLE
Release 0.9.8.1

### DISCUSSION THREAD

**Discussion by jon-harper** [jon-harper/OmniBox#30] [Announcements]
## Core

- Print time and material usage is slightly reduced for the front and rear base components.

## Panels and Trays

- Lid for filament spools added.
    - An adaptation of my [Tighter T.U.S.H. spool holder](https://www.thingiverse.com/thing:4737072) (a remix of the original T.U.S.H.).
    - It mounts on a lid that integrates the spool holder guide.
    - The guide makes the spool holder adjustable to spool width (up to about 120mm).
- Lid for dryboxes added.
    - Fitted for Rubbermaid 21 cup container used as a 1kg spool drybox.
    - These containers are about $10 each at [Ace Hardware](https://www.acehardware.com/departments/home-and-decor/kitchen-utensils-and-gadgets/food-storage/6192892). They are more expensive on [Amazon](https://www.amazon.com/Rubbermaid-LEPUSEMTE469-711717429496-Container-Everyday/dp/B00XJRMW5M).
    - There are often clones of these on Amazon that are similarly priced (about $8-10 ea).
    - May have lids for other containers in the near future as a user contribution.
- Raspberry Pi CPU trays have a version with a 6-pin Micro Fit 3 panel mount cutout.
    - Allow using an ADXL345 accelerometer via SPI.
    - Cutout is on the tray to give shortest wiring run possible.

### Experimental Hardware Support

These parts still need to be tested for fit and other issues.

| Hardware | Mount Location(s) | Notes |
|----------|-------------------|-------|
| Mean Well RSP-500 power supply | Main body | For the Ender 5 Plus |
| 92mm fans | Rear panel, lid | 25mm thickness; no compatible mount yet. |

### New Hardware Support

This hardware is now fit-tested as working.

| Hardware | Mount Location(s) | Notes |
|----------|-------------------|-------|
| Fotek SSR-60 DA solid state relay | Lower bay | |
| Raspberry Pi TFTs | Display mount or half-lid | See below. |

Note: Smaller TFTs (e.g., BigTreeTech's 5" PiTFT) will fit on the standard half-lid. Use the "XL" STL file for the official 7" and similarly large TFTs. 

## Documentation & Repository

- Documentation site
    - Modified color scheme and fonts for readability.
    - Added new dark color theme.
    - Implemented several Material for Mkdocs features, including icons to help with part identification.
    - Improved site navigation.
- Repository
    - Reorganized lids folder into "Full-Length" and "Half-Lenth" subfolders.
    - Added more readme.md files for folders.
## Known Issues

- The front main body with a 40mm intake fan partially obscures a mounting hole for the base to the main body. This leave seven (7) other mount points and is not a significant issue. The hole can safely be left empty.

<hr /><em>This discussion was created from the release <a href='https://github.com/jon-harper/OmniBox/releases/tag/0.9.8.1'>Release 0.9.8.1</a>.</em>

**Comment by TheoBromine2022** [comment]
Digging the new documentation!

as an aside, what is the channel on the rear base opposite from the power socket for? is it just a screw hole with a bend in it?

**Reply by jon-harper** [reply]
Thanks! I *really* wanted to make assembly and finding info easier.

Oh, yeah, that hole is for the rear panel. It only went in partway from the rear. I punched out that curve so that supports can be removed by pushing them through.
